### PR TITLE
Mate bumps

### PR DESCRIPTION
**Pull request by oz123**
These are the first in a row of version bumps for 1.26 series of mate-desktop.

### CODE REVIEW

File: dev-libs/libmateweather/libmateweather-1.26.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	x11-libs/gdk-pixbuf:2
+	>=x11-libs/gtk+-3.22:3
+"
+
+RDEPEND="${COMMON_DEPEND}
+	virtual/libintl
+"
+
+DEPEND="${COMMON_DEPEND}
+	dev-util/gtk-doc
```
Comment (thesamesam) [inline dev-libs/libmateweather/libmateweather-1.26.0.ebuild:33]: All of these are BDEPEND

File: mate-base/libmatekbd/libmatekbd-1.26.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	>=x11-libs/libxklavier-5.2:0[introspection?]
+	x11-libs/pango
+	>=x11-libs/gtk+-3.22:3[introspection?]
+	introspection? ( >=dev-libs/gobject-introspection-0.6.7:= )
+"
+
+RDEPEND="${COMMON_DEPEND}"
+
+DEPEND="${COMMON_DEPEND}
+	dev-libs/libxml2
```
Comment (thesamesam) [inline mate-base/libmatekbd/libmatekbd-1.26.0.ebuild:36]: BDEPENDs

File: mate-base/mate-desktop/mate-desktop-1.26.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	introspection? ( >=dev-libs/gobject-introspection-0.9.7:= )
+	startup-notification? ( >=x11-libs/startup-notification-0.5:0 )
+"
+
+RDEPEND="${COMMON_DEPEND}
+	virtual/libintl
+"
+
+DEPEND="${COMMON_DEPEND}
+	app-text/docbook-xml-dtd:4.1.2
```
Comment (thesamesam) [inline mate-base/mate-desktop/mate-desktop-1.26.0.ebuild:36]: all BDEPENDs apart from xorg-proto

File: mate-extra/mate-user-guide/mate-user-guide-1.26.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+if [[ ${PV} != 9999 ]]; then
+	KEYWORDS="~amd64 ~arm ~riscv ~x86"
+fi
+
+DESCRIPTION="User documentation for MATE Desktop"
+LICENSE="GPL-2 LGPL-2"
+SLOT="0"
+IUSE="nls"
+
+COMMON_DEPEND="virtual/libintl:0"
```
Comment (thesamesam) [inline mate-extra/mate-user-guide/mate-user-guide-1.26.0.ebuild:19]: drop :0

File: mate-extra/mate-user-guide/mate-user-guide-1.26.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+RDEPEND="${COMMON_DEPEND}
+	gnome-extra/yelp"
+
+DEPEND="${COMMON_DEPEND}
```
Comment (thesamesam) [inline mate-extra/mate-user-guide/mate-user-guide-1.26.0.ebuild:24]: all BDEPENDs and drop :* on pkgconfig

File: mate-base/mate-desktop/mate-desktop-1.26.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+	x11-libs/cairo
+	x11-libs/libX11
+	>=x11-libs/libXrandr-1.3
+	>=x11-libs/gtk+-3.22:3[introspection?]
+	introspection? ( >=dev-libs/gobject-introspection-0.9.7:= )
+	startup-notification? ( >=x11-libs/startup-notification-0.5:0 )
+"
+
+RDEPEND="${COMMON_DEPEND}
+	virtual/libintl
```
Comment (thesamesam) [inline mate-base/mate-desktop/mate-desktop-1.26.0.ebuild:32]: It's unlikely this belongs in only RDEPEND. It should be in COMMON_DEPEND. I don't think libintl is ever dlopen()'d.

Comment (oz123) [reply mate-base/mate-desktop/mate-desktop-1.26.0.ebuild]: Done.

File: mate-extra/mate-user-guide/mate-user-guide-1.26.0.ebuild (hunk 5 above)
Comment (oz123) [reply mate-extra/mate-user-guide/mate-user-guide-1.26.0.ebuild]: Done.

File: mate-base/libmatekbd/libmatekbd-1.26.0.ebuild (hunk 2 above)
Comment (oz123) [reply mate-base/libmatekbd/libmatekbd-1.26.0.ebuild]: Done.

File: dev-libs/libmateweather/libmateweather-1.26.0.ebuild (hunk 1 above)
Comment (oz123) [reply dev-libs/libmateweather/libmateweather-1.26.0.ebuild:33]: Thanks for the comperhensive review. I hope I understood you properly. I made the required changes and rebased all the commits.

Comment (thesamesam) [reply dev-libs/libmateweather/libmateweather-1.26.0.ebuild:33]: COMMON_DEPEND shouldn't be in BDEPEND here, I'll fix it

File: mate-base/libmatekbd/libmatekbd-1.26.0.ebuild [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+	x11-libs/libX11
+	>=x11-libs/libxklavier-5.2:0[introspection?]
+	x11-libs/pango
+	>=x11-libs/gtk+-3.22:3[introspection?]
+	introspection? ( >=dev-libs/gobject-introspection-0.6.7:= )
+"
+
+RDEPEND="${COMMON_DEPEND}"
+
+BDEPEND="${COMMON_DEPEND}
```
Comment (thesamesam) [inline mate-base/libmatekbd/libmatekbd-1.26.0.ebuild:35]: COMMON_DEPEND shouldn't be in BDEPEND here, I'll fix it

File: mate-base/mate-desktop/mate-desktop-1.26.0.ebuild [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+	>=gnome-base/dconf-0.13.4
+	x11-libs/cairo
+	x11-libs/libX11
+	>=x11-libs/libXrandr-1.3
+	>=x11-libs/gtk+-3.22:3[introspection?]
+	introspection? ( >=dev-libs/gobject-introspection-0.9.7:= )
+	startup-notification? ( >=x11-libs/startup-notification-0.5:0 )
+"
+
+BDEPEND="${COMMON_DEPEND}
```
Comment (thesamesam) [inline mate-base/mate-desktop/mate-desktop-1.26.0.ebuild:31]: COMMON_DEPEND shouldn't be in BDEPEND here, I'll fix it

File: mate-extra/mate-user-guide/mate-user-guide-1.26.0.ebuild [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+LICENSE="GPL-2 LGPL-2"
+SLOT="0"
+IUSE="nls"
+
+COMMON_DEPEND="virtual/libintl"
+
+RDEPEND="${COMMON_DEPEND}
+	gnome-extra/yelp"
+
+BDEPEND="${COMMON_DEPEND}
```
Comment (thesamesam) [inline mate-extra/mate-user-guide/mate-user-guide-1.26.0.ebuild:24]: COMMON_DEPEND shouldn't be in BDEPEND here, I'll fix it

File: mate-base/mate-desktop/mate-desktop-1.26.0.ebuild [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+	>=dev-libs/glib-2.50:2
+	>=gnome-base/dconf-0.13.4
+	x11-libs/cairo
+	x11-libs/libX11
+	>=x11-libs/libXrandr-1.3
+	>=x11-libs/gtk+-3.22:3[introspection?]
+	introspection? ( >=dev-libs/gobject-introspection-0.9.7:= )
+	startup-notification? ( >=x11-libs/startup-notification-0.5:0 )
+"
+
```
Comment (thesamesam) [inline mate-base/mate-desktop/mate-desktop-1.26.0.ebuild:30]: RDEPEND is blank here. I've added it to be equal to COMMON_DEPEND.

**Comment by thesamesam** [comment]
Thanks!

Please look at my fixups and ask me if you have questions:
- 6bc9ffbd57449188864e88410e51fbfe7da3f39f
- 7e271e6b59e1c1b786a78710fc34ee53724ddd10
- bdf0fc0b4f26fbb432031d352f69142792918972
- 7ea1c7142f757c247f56f07355f3431d01644697
- 35fdbe3d7937a6c3daca044fdc616982a18b7bad

**Pull request closed without merging**